### DISCUSSION TITLE
[Feature] Drag'n Drop for Photos in Album

### DISCUSSION THREAD

**Discussion by MickLesk** [immich-app/immich#8744] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

IHello dear development team,

In connection with my other feature request --> https://github.com/immich-app/immich/discussions/8743
it would be conceivable that I am on the Immich page, e.g. click (select) 5 photos and drag'n drop them into the album in the sidebar.
=> This adds them to the album.

Logic:
- It must be checked whether this photo is already in the album, if so skip with note
- There should be a feedback dialog that the elements have been moved successfully (or not) (similar to the upload dialog) 

Unfortunately I can't fake this somehow, I hope it's understandable, otherwise I'll have to build a "click dummy" on a developer environment :-) 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by mertalev** [comment]
This is a cool idea actually. I can't implement it because I'm not a frontend dev, but a drag-and-drop would be very intuitive here.
